examples/platform/silabs/SiWx917/SiWx917/sl_wifi_if.c

```diff
@@ -268,13 +265,18 @@ static int32_t wfx_rsi_init(void)
 }
 
 /*************************************************************************************
- * @fn  sl_status_t scan_callback_handler
+ * @fn  void wfx_show_err(char *msg)
  * @brief
- *      scan callback handler
+ *      driver shows error message
  * @param[in]  msg
  * @return
- *        sl_status_t
+ *        None
  *****************************************************************************************/
+void wfx_show_err(char * msg)
+{
+    SILABS_LOG("%s: message: %d", __func__, msg);
```

Comment:
I left a comment about this last time, Seems like it is gone. We advocate for not using __func__ in all traces. It doesn't bring any value but code size increase